node/flatpak-node-generator.py

```diff
@@ -66,6 +66,7 @@
 
 NPM_MIRROR = 'https://unpkg.com/'
 
+NPM_CORGIDOC = 'application/vnd.npm.install-v1+json; q=1.0, application/json; q=0.8, */*'
```

Comment:
Could you add a comment describing what this is / where it's from? I...honestly can';t find anything on a "corgidoc" header, unless my search-fu is just failing me :sweat_smile: (Unless it's just magic taken from somewhere in the npm source code, in which case...oof)